scripts/run_docker

```diff
@@ -3,7 +3,7 @@
 cd "$(dirname "$0")" || exit 1
 CONTAINER_RUNTIME="${CONTAINER_RUNTIME:-docker}"
 
-$CONTAINER_RUNTIME build -t aries-cloudagent-run -f ../docker/Dockerfile.run .. || exit 1
+$CONTAINER_RUNTIME build --platform linux/amd64 -t aries-cloudagent-run -f ../docker/Dockerfile.run .. || exit 1
```

Comment:
This looks like it was adjusted for the dev's system? It could be useful to support an environment variable for setting the platform, though.

Reply:
Indeed, this was added to support running the script on M1, in the absence of support for images specifically built for aarch64.